Sources/Types+Media.swift

```diff
@@ -95,7 +96,7 @@ extension Media {
     public var bitRate: Int?
     
     /// Type of media
-    public var contentType: MediaType
+    public var contentType: UTType
```

Comment:
As mentioned in my comment, this should actually probably conform to simply `String` if we can't decode `UTType` from it (which is irritating, since `UTType(mimeType: String)` would give us the desired result)
```suggestion
    public var contentType: String
```